test/test_rospkg_catkin_packages.py

```diff
@@ -54,7 +54,13 @@ def test_find_packages():
 
 
 def test_get_manifest():
-    search_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'catkin_package_tests'))
     manager = rospkg.rospack.ManifestManager(rospkg.common.MANIFEST_FILE, ros_paths=[search_path])
     manif = manager.get_manifest("foo")
     assert(manif.type == "package")
+
+
+def test_licenses():
+    manager = rospkg.rospack.RosPack(ros_paths=[search_path])
+    manif = manager.get_manifest("foo")
+    assert(manif.license == "BSD, LGPL")
+    assert(len(manif.licenses) == 2)  # package.xml in 'foo' defines BSD and LGPL
```

Comment:
Instead of mentioning it in the comment please check both license makes explicitly with `assert`.